podpac/core/node.py

```diff
@@ -95,17 +104,39 @@ class Node(tl.HasTraits):
 
     units = Units(default_value=None, allow_none=True)
     dtype = tl.Any(default_value=float)
+    cache_output = tl.Bool(True)
```

Comment:
Presumably you don't always want to cache the output for all nodes. YOu can choose not to decorate the `eval` function for classes of `Nodes` but this gives you instance-level control.